tests/browser-detector/scripts/build.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+error_reporting(E_ERROR | E_WARNING | E_PARSE);
+
+$allTests = [];
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$jsonParser = new \Seld\JsonLint\JsonParser();
+
+$finder = new \Symfony\Component\Finder\Finder();
+$finder->files();
+$finder->name('*.json');
+$finder->ignoreDotFiles(true);
+$finder->ignoreVCS(true);
+$finder->sortByName();
+$finder->ignoreUnreadableDirs();
+$finder->in(__DIR__ . '/../vendor/mimmi20/browser-detector-tests/tests/issues');
+
+foreach ($finder as $file) {
+    /** @var \Symfony\Component\Finder\SplFileInfo $file */
+    if (!$file->isFile() || 'json' !== $file->getExtension()) {
```

Comment:
This shouldn't be necessary no?  The Finder should be limiting this.

Reply:
fixed